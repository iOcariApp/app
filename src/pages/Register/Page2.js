import React from "react";
import PropTypes from "prop-types";
import { ScrollView } from "react-native";
import styles from "./Register.style";

import TextInput from "components/TextInput";
import DualRow from "components/DualRow";
import Button from "components/Button";
import WhiteEmptyButton from "components/Button/WhiteEmptyButton";

const Page2 = ({
  values: { name, surname, birthdate, address, country, city, postalCode },
  logic: {
    onChangeName,
    onChangeSurname,
    onChangeBirthdate,
    onChangeAddress,
    onChangeCountry,
    onChangeCity,
    onChangePostalCode,
    validValidation,
    goPrev,
    goNext,
  },
}) => (
  <ScrollView style={styles.containerScrollable}>
    <TextInput
      label="Nombre"
      value={name}
      onChangeValue={onChangeName}
      validation={validValidation}
    />
    <TextInput
      label="Apellido"
      value={surname}
      onChangeValue={onChangeSurname}
      validation={validValidation}
    />
    <TextInput
      label="Fecha de nacimiento"
      value={birthdate}
      onChangeValue={onChangeBirthdate}
      validation={validValidation}
    />
    <TextInput
      label="Dirección"
      value={address}
      onChangeValue={onChangeAddress}
      validation={validValidation}
    />
    <TextInput
      label="País"
      value={country}
      onChangeValue={onChangeCountry}
      validation={validValidation}
    />
    <DualRow
      left={
        <TextInput
          label="Ciudad"
          value={city}
          onChangeValue={onChangeCity}
          validation={validValidation}
        />
      }
      right={
        <TextInput
          label="Código Postal"
          value={postalCode}
          onChangeValue={onChangePostalCode}
          validation={validValidation}
        />
      }
      separation={15}
    />
    <DualRow
      style={styles.marginButtons}
      left={<WhiteEmptyButton title="ATRÁS" onPress={goPrev} />}
      right={<Button title="¡LISTO!" onPress={goNext} />}
      separation={7}
    />
  </ScrollView>
);

Page2.propTypes = {
  values: PropTypes.shape({
    name: PropTypes.string,
    surname: PropTypes.string,
    birthdate: PropTypes.string,
    address: PropTypes.string,
    country: PropTypes.string,
    city: PropTypes.string,
    postalCode: PropTypes.string,
  }),
  logic: PropTypes.shape({
    onChangeName: PropTypes.func,
    onChangeSurname: PropTypes.func,
    onChangeBirthdate: PropTypes.func,
    onChangeAddress: PropTypes.func,
    onChangeCountry: PropTypes.func,
    onChangeCity: PropTypes.func,
    onChangePostalCode: PropTypes.func,
    validValidation: PropTypes.func,
    goPrev: PropTypes.func,
    goNext: PropTypes.func,
  }),
};

export default Page2;
