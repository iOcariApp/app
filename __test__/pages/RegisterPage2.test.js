import React from "react";

import Page2 from "pages/Register/Page2";

const props = {
  values: {
    name: "",
    surname: "",
    birthdate: "",
    address: "",
    country: "",
    city: "",
    postalCode: "",
  },
  onChangeValue: noop,
  goPrev: noop,
  showPrivacyPopup: noop,
  showCalendarBirthdate: noop,
};

test("renders correctly", () => {
  testSnapshot(<Page2 {...props} />);
});
