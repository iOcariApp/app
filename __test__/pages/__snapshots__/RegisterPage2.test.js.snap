// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<RCTScrollView
  style={
    Object {
      "marginTop": 20,
      "width": "100%",
    }
  }
>
  <View>
    <View>
      <View
        onLayout={[Function]}
        style={
          Object {
            "marginBottom": 17,
            "width": "100%",
          }
        }
      >
        <View
          style={
            Object {
              "flexDirection": "column",
              "justifyContent": "center",
              "position": "relative",
              "width": "100%",
            }
          }
        >
          <View
            accessibilityComponentType={undefined}
            accessibilityLabel={undefined}
            accessibilityTraits={undefined}
            accessible={true}
            data-test="input-main"
            hitSlop={undefined}
            nativeID={undefined}
            onLayout={undefined}
            onResponderGrant={[Function]}
            onResponderMove={[Function]}
            onResponderRelease={[Function]}
            onResponderTerminate={[Function]}
            onResponderTerminationRequest={[Function]}
            onStartShouldSetResponder={[Function]}
            style={
              Object {
                "alignContent": "center",
                "backgroundColor": "white",
                "borderBottomWidth": 1,
                "borderColor": "#7C7C7C",
                "borderTopLeftRadius": 4,
                "borderTopRightRadius": 4,
                "flexDirection": "row",
                "height": 56,
                "paddingBottom": 10,
                "paddingLeft": 15,
                "paddingRight": 15,
                "paddingTop": 10,
                "width": "100%",
              }
            }
            testID={undefined}
          >
            <View
              style={
                Object {
                  "flex": 1,
                  "position": "relative",
                  "top": -7,
                }
              }
            >
              <Text
                accessible={true}
                allowFontScaling={true}
                collapsable={undefined}
                ellipsizeMode="tail"
                style={
                  Object {
                    "color": "#7C7C7C",
                    "fontSize": 16,
                    "position": "relative",
                    "top": 14,
                  }
                }
              >
                Nombre
              </Text>
              <TextInput
                allowFontScaling={true}
                keyLabel="name"
                onBlur={[Function]}
                onChangeText={[Function]}
                onChangeValue={[MockFunction]}
                onFocus={[Function]}
                onSubmitEditing={[Function]}
                returnKeyType="next"
                style={
                  Object {
                    "fontSize": 16,
                  }
                }
                underlineColorAndroid="transparent"
                validation={[Function]}
                value=""
              />
            </View>
          </View>
        </View>
      </View>
      <View
        onLayout={[Function]}
        style={
          Object {
            "marginBottom": 17,
            "width": "100%",
          }
        }
      >
        <View
          style={
            Object {
              "flexDirection": "column",
              "justifyContent": "center",
              "position": "relative",
              "width": "100%",
            }
          }
        >
          <View
            accessibilityComponentType={undefined}
            accessibilityLabel={undefined}
            accessibilityTraits={undefined}
            accessible={true}
            data-test="input-main"
            hitSlop={undefined}
            nativeID={undefined}
            onLayout={undefined}
            onResponderGrant={[Function]}
            onResponderMove={[Function]}
            onResponderRelease={[Function]}
            onResponderTerminate={[Function]}
            onResponderTerminationRequest={[Function]}
            onStartShouldSetResponder={[Function]}
            style={
              Object {
                "alignContent": "center",
                "backgroundColor": "white",
                "borderBottomWidth": 1,
                "borderColor": "#7C7C7C",
                "borderTopLeftRadius": 4,
                "borderTopRightRadius": 4,
                "flexDirection": "row",
                "height": 56,
                "paddingBottom": 10,
                "paddingLeft": 15,
                "paddingRight": 15,
                "paddingTop": 10,
                "width": "100%",
              }
            }
            testID={undefined}
          >
            <View
              style={
                Object {
                  "flex": 1,
                  "position": "relative",
                  "top": -7,
                }
              }
            >
              <Text
                accessible={true}
                allowFontScaling={true}
                collapsable={undefined}
                ellipsizeMode="tail"
                style={
                  Object {
                    "color": "#7C7C7C",
                    "fontSize": 16,
                    "position": "relative",
                    "top": 14,
                  }
                }
              >
                Apellido
              </Text>
              <TextInput
                allowFontScaling={true}
                keyLabel="surname"
                onBlur={[Function]}
                onChangeText={[Function]}
                onChangeValue={[MockFunction]}
                onFocus={[Function]}
                onSubmitEditing={[Function]}
                returnKeyType="next"
                style={
                  Object {
                    "fontSize": 16,
                  }
                }
                underlineColorAndroid="transparent"
                validation={[Function]}
                value=""
              />
            </View>
          </View>
        </View>
      </View>
      <View
        onLayout={[Function]}
        style={
          Object {
            "marginBottom": 17,
            "width": "100%",
          }
        }
      >
        <View
          style={
            Object {
              "flexDirection": "column",
              "justifyContent": "center",
              "position": "relative",
              "width": "100%",
            }
          }
        >
          <View
            accessibilityComponentType={undefined}
            accessibilityLabel={undefined}
            accessibilityTraits={undefined}
            accessible={true}
            data-test="input-main"
            hitSlop={undefined}
            nativeID={undefined}
            onLayout={undefined}
            onResponderGrant={[Function]}
            onResponderMove={[Function]}
            onResponderRelease={[Function]}
            onResponderTerminate={[Function]}
            onResponderTerminationRequest={[Function]}
            onStartShouldSetResponder={[Function]}
            style={
              Object {
                "alignContent": "center",
                "backgroundColor": "white",
                "borderBottomWidth": 1,
                "borderColor": "#7C7C7C",
                "borderTopLeftRadius": 4,
                "borderTopRightRadius": 4,
                "flexDirection": "row",
                "height": 56,
                "paddingBottom": 10,
                "paddingLeft": 15,
                "paddingRight": 15,
                "paddingTop": 10,
                "width": "100%",
              }
            }
            testID={undefined}
          >
            <View
              style={
                Object {
                  "flex": 1,
                  "position": "relative",
                  "top": -7,
                }
              }
            >
              <Text
                accessible={true}
                allowFontScaling={true}
                collapsable={undefined}
                ellipsizeMode="tail"
                style={
                  Object {
                    "color": "#7C7C7C",
                    "fontSize": 16,
                    "position": "relative",
                    "top": 14,
                  }
                }
              >
                Fecha de nacimiento
              </Text>
              <TextInput
                allowFontScaling={true}
                keyLabel="birthdate"
                onBlur={[Function]}
                onChangeText={[Function]}
                onChangeValue={[MockFunction]}
                onClick={[MockFunction]}
                onFocus={[Function]}
                onSubmitEditing={[Function]}
                returnKeyType="next"
                style={
                  Object {
                    "fontSize": 16,
                  }
                }
                underlineColorAndroid="transparent"
                validation={[Function]}
                value=""
              />
            </View>
          </View>
        </View>
      </View>
      <View
        onLayout={[Function]}
        style={
          Object {
            "marginBottom": 17,
            "width": "100%",
          }
        }
      >
        <View
          style={
            Object {
              "flexDirection": "column",
              "justifyContent": "center",
              "position": "relative",
              "width": "100%",
            }
          }
        >
          <View
            accessibilityComponentType={undefined}
            accessibilityLabel={undefined}
            accessibilityTraits={undefined}
            accessible={true}
            data-test="input-main"
            hitSlop={undefined}
            nativeID={undefined}
            onLayout={undefined}
            onResponderGrant={[Function]}
            onResponderMove={[Function]}
            onResponderRelease={[Function]}
            onResponderTerminate={[Function]}
            onResponderTerminationRequest={[Function]}
            onStartShouldSetResponder={[Function]}
            style={
              Object {
                "alignContent": "center",
                "backgroundColor": "white",
                "borderBottomWidth": 1,
                "borderColor": "#7C7C7C",
                "borderTopLeftRadius": 4,
                "borderTopRightRadius": 4,
                "flexDirection": "row",
                "height": 56,
                "paddingBottom": 10,
                "paddingLeft": 15,
                "paddingRight": 15,
                "paddingTop": 10,
                "width": "100%",
              }
            }
            testID={undefined}
          >
            <View
              style={
                Object {
                  "flex": 1,
                  "position": "relative",
                  "top": -7,
                }
              }
            >
              <Text
                accessible={true}
                allowFontScaling={true}
                collapsable={undefined}
                ellipsizeMode="tail"
                style={
                  Object {
                    "color": "#7C7C7C",
                    "fontSize": 16,
                    "position": "relative",
                    "top": 14,
                  }
                }
              >
                Dirección
              </Text>
              <TextInput
                allowFontScaling={true}
                keyLabel="address"
                onBlur={[Function]}
                onChangeText={[Function]}
                onChangeValue={[MockFunction]}
                onFocus={[Function]}
                onSubmitEditing={[Function]}
                returnKeyType="next"
                style={
                  Object {
                    "fontSize": 16,
                  }
                }
                underlineColorAndroid="transparent"
                validation={[Function]}
                value=""
              />
            </View>
          </View>
        </View>
      </View>
      <View
        onLayout={[Function]}
        style={
          Object {
            "marginBottom": 17,
            "width": "100%",
          }
        }
      >
        <View
          style={
            Object {
              "flexDirection": "column",
              "justifyContent": "center",
              "position": "relative",
              "width": "100%",
            }
          }
        >
          <View
            accessibilityComponentType={undefined}
            accessibilityLabel={undefined}
            accessibilityTraits={undefined}
            accessible={true}
            data-test="input-main"
            hitSlop={undefined}
            nativeID={undefined}
            onLayout={undefined}
            onResponderGrant={[Function]}
            onResponderMove={[Function]}
            onResponderRelease={[Function]}
            onResponderTerminate={[Function]}
            onResponderTerminationRequest={[Function]}
            onStartShouldSetResponder={[Function]}
            style={
              Object {
                "alignContent": "center",
                "backgroundColor": "white",
                "borderBottomWidth": 1,
                "borderColor": "#7C7C7C",
                "borderTopLeftRadius": 4,
                "borderTopRightRadius": 4,
                "flexDirection": "row",
                "height": 56,
                "paddingBottom": 10,
                "paddingLeft": 15,
                "paddingRight": 15,
                "paddingTop": 10,
                "width": "100%",
              }
            }
            testID={undefined}
          >
            <View
              style={
                Object {
                  "flex": 1,
                  "position": "relative",
                  "top": -7,
                }
              }
            >
              <Text
                accessible={true}
                allowFontScaling={true}
                collapsable={undefined}
                ellipsizeMode="tail"
                style={
                  Object {
                    "color": "#7C7C7C",
                    "fontSize": 16,
                    "position": "relative",
                    "top": 14,
                  }
                }
              >
                País
              </Text>
              <TextInput
                allowFontScaling={true}
                keyLabel="country"
                onBlur={[Function]}
                onChangeText={[Function]}
                onChangeValue={[MockFunction]}
                onFocus={[Function]}
                onSubmitEditing={[Function]}
                returnKeyType="next"
                style={
                  Object {
                    "fontSize": 16,
                  }
                }
                underlineColorAndroid="transparent"
                validation={[Function]}
                value=""
              />
            </View>
          </View>
        </View>
      </View>
      <View
        style={
          Array [
            Object {
              "alignItems": "flex-start",
              "flexDirection": "row",
              "justifyContent": "space-between",
              "width": "100%",
            },
            Object {},
          ]
        }
      >
        <View
          style={
            Array [
              Object {
                "flex": 1,
              },
              Object {
                "marginRight": 15,
              },
            ]
          }
        >
          <View
            onLayout={[Function]}
            style={
              Object {
                "marginBottom": 17,
                "width": "100%",
              }
            }
          >
            <View
              style={
                Object {
                  "flexDirection": "column",
                  "justifyContent": "center",
                  "position": "relative",
                  "width": "100%",
                }
              }
            >
              <View
                accessibilityComponentType={undefined}
                accessibilityLabel={undefined}
                accessibilityTraits={undefined}
                accessible={true}
                data-test="input-main"
                hitSlop={undefined}
                nativeID={undefined}
                onLayout={undefined}
                onResponderGrant={[Function]}
                onResponderMove={[Function]}
                onResponderRelease={[Function]}
                onResponderTerminate={[Function]}
                onResponderTerminationRequest={[Function]}
                onStartShouldSetResponder={[Function]}
                style={
                  Object {
                    "alignContent": "center",
                    "backgroundColor": "white",
                    "borderBottomWidth": 1,
                    "borderColor": "#7C7C7C",
                    "borderTopLeftRadius": 4,
                    "borderTopRightRadius": 4,
                    "flexDirection": "row",
                    "height": 56,
                    "paddingBottom": 10,
                    "paddingLeft": 15,
                    "paddingRight": 15,
                    "paddingTop": 10,
                    "width": "100%",
                  }
                }
                testID={undefined}
              >
                <View
                  style={
                    Object {
                      "flex": 1,
                      "position": "relative",
                      "top": -7,
                    }
                  }
                >
                  <Text
                    accessible={true}
                    allowFontScaling={true}
                    collapsable={undefined}
                    ellipsizeMode="tail"
                    style={
                      Object {
                        "color": "#7C7C7C",
                        "fontSize": 16,
                        "position": "relative",
                        "top": 14,
                      }
                    }
                  >
                    Ciudad
                  </Text>
                  <TextInput
                    allowFontScaling={true}
                    keyLabel="city"
                    onBlur={[Function]}
                    onChangeText={[Function]}
                    onChangeValue={[MockFunction]}
                    onFocus={[Function]}
                    onSubmitEditing={[Function]}
                    returnKeyType="next"
                    style={
                      Object {
                        "fontSize": 16,
                      }
                    }
                    underlineColorAndroid="transparent"
                    validation={[Function]}
                    value=""
                  />
                </View>
              </View>
            </View>
          </View>
        </View>
        <View
          style={
            Object {
              "flex": 1,
            }
          }
        >
          <View
            onLayout={[Function]}
            style={
              Object {
                "marginBottom": 17,
                "width": "100%",
              }
            }
          >
            <View
              style={
                Object {
                  "flexDirection": "column",
                  "justifyContent": "center",
                  "position": "relative",
                  "width": "100%",
                }
              }
            >
              <View
                accessibilityComponentType={undefined}
                accessibilityLabel={undefined}
                accessibilityTraits={undefined}
                accessible={true}
                data-test="input-main"
                hitSlop={undefined}
                nativeID={undefined}
                onLayout={undefined}
                onResponderGrant={[Function]}
                onResponderMove={[Function]}
                onResponderRelease={[Function]}
                onResponderTerminate={[Function]}
                onResponderTerminationRequest={[Function]}
                onStartShouldSetResponder={[Function]}
                style={
                  Object {
                    "alignContent": "center",
                    "backgroundColor": "white",
                    "borderBottomWidth": 1,
                    "borderColor": "#7C7C7C",
                    "borderTopLeftRadius": 4,
                    "borderTopRightRadius": 4,
                    "flexDirection": "row",
                    "height": 56,
                    "paddingBottom": 10,
                    "paddingLeft": 15,
                    "paddingRight": 15,
                    "paddingTop": 10,
                    "width": "100%",
                  }
                }
                testID={undefined}
              >
                <View
                  style={
                    Object {
                      "flex": 1,
                      "position": "relative",
                      "top": -7,
                    }
                  }
                >
                  <Text
                    accessible={true}
                    allowFontScaling={true}
                    collapsable={undefined}
                    ellipsizeMode="tail"
                    style={
                      Object {
                        "color": "#7C7C7C",
                        "fontSize": 16,
                        "position": "relative",
                        "top": 14,
                      }
                    }
                  >
                    Código Postal
                  </Text>
                  <TextInput
                    allowFontScaling={true}
                    keyLabel="postalCode"
                    onBlur={[Function]}
                    onChangeText={[Function]}
                    onChangeValue={[MockFunction]}
                    onFocus={[Function]}
                    onSubmitEditing={[Function]}
                    returnKeyType="done"
                    style={
                      Object {
                        "fontSize": 16,
                      }
                    }
                    underlineColorAndroid="transparent"
                    validation={[Function]}
                    value=""
                  />
                </View>
              </View>
            </View>
          </View>
        </View>
      </View>
      <View
        style={
          Array [
            Object {
              "alignItems": "flex-start",
              "flexDirection": "row",
              "justifyContent": "space-between",
              "width": "100%",
            },
            Object {
              "marginBottom": 20,
              "marginTop": 20,
            },
          ]
        }
      >
        <View
          style={
            Array [
              Object {
                "flex": 1,
              },
              Object {
                "marginRight": 7,
              },
            ]
          }
        >
          <View
            style={
              Array [
                Object {
                  "marginLeft": 15,
                  "marginRight": 15,
                },
                undefined,
                Array [
                  Object {
                    "marginBottom": 0,
                    "marginLeft": 0,
                    "marginRight": 0,
                    "marginTop": 0,
                    "width": "100%",
                  },
                  Object {},
                ],
                Object {
                  "borderRadius": 4,
                },
              ]
            }
          >
            <View
              accessibilityComponentType={undefined}
              accessibilityLabel={undefined}
              accessibilityTraits={undefined}
              accessible={true}
              hasTVPreferredFocus={undefined}
              hitSlop={undefined}
              isTVSelectable={true}
              nativeID={undefined}
              onLayout={undefined}
              onResponderGrant={[Function]}
              onResponderMove={[Function]}
              onResponderRelease={[Function]}
              onResponderTerminate={[Function]}
              onResponderTerminationRequest={[Function]}
              onStartShouldSetResponder={[Function]}
              style={null}
              testID={undefined}
              tvParallaxProperties={undefined}
            >
              <View
                pointerEvents="box-only"
                style={
                  Array [
                    Object {
                      "alignItems": "center",
                      "backgroundColor": "#9E9E9E",
                      "flexDirection": "row",
                      "justifyContent": "center",
                      "padding": 19,
                    },
                    undefined,
                    undefined,
                    undefined,
                    undefined,
                    undefined,
                    Object {
                      "backgroundColor": "transparent",
                    },
                    Object {
                      "borderRadius": 4,
                    },
                    Object {
                      "padding": 12,
                    },
                    undefined,
                    undefined,
                    undefined,
                    Array [
                      Object {
                        "height": 56,
                      },
                      Object {
                        "borderColor": "white",
                        "borderWidth": 2,
                      },
                    ],
                    undefined,
                    undefined,
                  ]
                }
              >
                <Text
                  accessible={true}
                  allowFontScaling={true}
                  ellipsizeMode="tail"
                  style={
                    Array [
                      Object {},
                      undefined,
                      undefined,
                      undefined,
                      undefined,
                      undefined,
                      undefined,
                      undefined,
                      undefined,
                      undefined,
                      Array [
                        Object {
                          "color": "white",
                          "fontSize": 20,
                        },
                        Object {
                          "color": "white",
                        },
                        Object {
                          "fontSize": 17.5,
                        },
                        Object {
                          "fontSize": 18,
                        },
                        undefined,
                        undefined,
                        undefined,
                        undefined,
                      ],
                    ]
                  }
                >
                  ATRÁS
                </Text>
              </View>
            </View>
          </View>
        </View>
        <View
          style={
            Object {
              "flex": 1,
            }
          }
        >
          <View
            style={
              Array [
                Object {
                  "marginLeft": 15,
                  "marginRight": 15,
                },
                undefined,
                Array [
                  Object {
                    "marginBottom": 0,
                    "marginLeft": 0,
                    "marginRight": 0,
                    "marginTop": 0,
                    "width": "100%",
                  },
                  Object {},
                ],
                Object {
                  "borderRadius": 4,
                },
              ]
            }
          >
            <View
              accessibilityComponentType={undefined}
              accessibilityLabel={undefined}
              accessibilityTraits={undefined}
              accessible={true}
              hasTVPreferredFocus={undefined}
              hitSlop={undefined}
              isTVSelectable={true}
              nativeID={undefined}
              onLayout={undefined}
              onResponderGrant={[Function]}
              onResponderMove={[Function]}
              onResponderRelease={[Function]}
              onResponderTerminate={[Function]}
              onResponderTerminationRequest={[Function]}
              onStartShouldSetResponder={[Function]}
              style={null}
              testID={undefined}
              tvParallaxProperties={undefined}
            >
              <View
                pointerEvents="box-only"
                style={
                  Array [
                    Object {
                      "alignItems": "center",
                      "backgroundColor": "#9E9E9E",
                      "flexDirection": "row",
                      "justifyContent": "center",
                      "padding": 19,
                    },
                    undefined,
                    undefined,
                    undefined,
                    undefined,
                    undefined,
                    Object {
                      "backgroundColor": "#F50057",
                    },
                    Object {
                      "borderRadius": 4,
                    },
                    Object {
                      "padding": 12,
                    },
                    undefined,
                    undefined,
                    undefined,
                    Array [
                      Object {
                        "height": 56,
                      },
                      Object {},
                    ],
                    undefined,
                    undefined,
                  ]
                }
              >
                <Text
                  accessible={true}
                  allowFontScaling={true}
                  ellipsizeMode="tail"
                  style={
                    Array [
                      Object {},
                      undefined,
                      undefined,
                      undefined,
                      undefined,
                      undefined,
                      undefined,
                      undefined,
                      undefined,
                      undefined,
                      Array [
                        Object {
                          "color": "white",
                          "fontSize": 20,
                        },
                        undefined,
                        Object {
                          "fontSize": 17.5,
                        },
                        Object {
                          "fontSize": 18,
                        },
                        undefined,
                        undefined,
                        undefined,
                        undefined,
                      ],
                    ]
                  }
                >
                  ¡LISTO!
                </Text>
              </View>
            </View>
          </View>
        </View>
      </View>
    </View>
  </View>
</RCTScrollView>
`;
